### PR TITLE
Create a more modular approach for all debian based systems

### PR DESCRIPTION
**Pull request by gonuke**
This has been tested in Ubuntu 16, but not anywhere else

### CODE REVIEW

**Comment by gonuke** [comment]
Anyone interested in reviewing this? @RLReed @kkiesling @elliottbiondo

**Comment by scopatz** [comment]
Looks good to me!  I haven't run it though...

File: ubuntu_mint.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    export LD_LIBRARY_PATH=$HOME/.local/lib:$LD_LIBRARY_PATH
+    ./scripts/nuc_data_make
+    
+}
+
+function test_pyne {
+
+    # Run all the tests
+    cd tests
+    . ./travis-run-tests.sh
```
Comment (gonuke) [inline ubuntu_mint.sh]: The first `.` is a shorthand for `source` in a bash shell

Comment (gonuke) [inline ubuntu_mint.sh]: I changed it for clarity, though

**Pull request merged**